src/pages/index.js

```diff
@@ -2,27 +2,9 @@ import React from "react"
 import { graphql } from "gatsby"
 import PropTypes from "prop-types"
 
-import { normalizeSEO } from "../helpers"
+import Page from "../templates/page/adapter"
```

Comment:
I had realized that it was unnecessary to duplicate all this code and I could simply pass the home page's query result onto the Page template adapter and let it do all the work.